frontend/src/metabase/entities/tables.js

```diff
@@ -21,7 +21,7 @@ import Metrics from "metabase/entities/metrics";
 import Segments from "metabase/entities/segments";
 import Questions from "metabase/entities/questions";
 
-import { GET, PUT } from "metabase/lib/api";
+import { PUT } from "metabase/lib/api";
```

Comment:
Changes in this file are unrelated to the PR.
I reused existing `MetabaseApi` definitions of endpoints.
There should be no difference in how anything works here.